bp_common/src/include/bp_common_aviary_defines.vh

```diff
@@ -117,6 +117,7 @@ typedef struct packed
   integer acache_assoc;
   integer acache_block_width;
   integer acache_fill_width;
+  integer cce_mem_if_data_width;
```

Comment:
I don't think we need this as a parameter at the top level.  It's really a derivative system property depending on adapter configuration, etc